src/main/java/bio/terra/workspace/service/workspace/flight/CreateControlledResourceFlight.java

```diff
@@ -0,0 +1,57 @@
+package bio.terra.workspace.service.workspace.flight;
```

Comment:
I think this flight and its steps should be in `bio.terra.workspace.service.resource...`, not the workspace package

Reply:
good point